policies/authorization_grant.rego

```diff
@@ -20,6 +20,7 @@ allowed_scope("email") = true
 
 # This grants access to Synapse's admin API endpoints
 allowed_scope("urn:synapse:admin:*") {
+	input.grant_type == "authorization_code"
```

Comment:
Synapse doesn't support user-less tokens yet, so we don't grant access to the admin API through the `client_credentials` for now

Reply:
It is probably redundant, but anyway, once Synapse has support for user-less tokens, the logic will branch out depending on whether it is a authorization_code grant or a client_credentials one, like I did on the `urn:mas:admin` scope